src/import.rs

```diff
@@ -113,12 +113,10 @@ fn resolve(import: Import, store: &mut Store) -> ExternVal {
         "literal_debug" => Box::new(sym::literal_debug),
         "span_debug" => Box::new(sym::span_debug),
 
-        "watt_string_with_capacity" => Box::new(sym::watt_string_with_capacity),
-        "watt_string_push_char" => Box::new(sym::watt_string_push_char),
+        "watt_string_new" => Box::new(sym::watt_string_new),
         "watt_string_len" => Box::new(sym::watt_string_len),
-        "watt_string_char_at" => Box::new(sym::watt_string_char_at),
-        "watt_bytes_with_capacity" => Box::new(sym::watt_bytes_with_capacity),
-        "watt_bytes_push" => Box::new(sym::watt_bytes_push),
```

Comment:
Occurs to me it might be worth keeping the old methods around if someone uses a pre-built wasm module with the new runtime.

Reply:
For now I've added these back in https://github.com/dtolnay/watt/commit/57d30e71f1309458e45fc7746ab3607c6f3f4b8d. We can drop them when releasing 0.2.